_releases/9.0.0.md

```diff
@@ -7,9 +7,9 @@ date: 2020-05-11
 summary: >
     Release v9.0.0
 
-artifact-root: "https://archive.apache.org/dist/incubator/nuttx/9.0.0"
```

Comment:
The archive links should _not_ be changed. Only the download ones.

Reply:
Done.